cli/compose/loader/types_test.go

```diff
@@ -0,0 +1,328 @@
+package loader
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func TestMarshallConfig(t *testing.T) {
+	cfg := fullExampleConfig("/foo", "/bar")
+	expected := `configs: {}
+networks:
```

Comment:
Hopefully `MarshalYAML` is stable and doesn't change order randomly when we change fields.

Reply:
so far it did not 😇